examples/chat/client/tasks.nim

```diff
@@ -633,3 +633,99 @@ proc publishWakuChat2*(message: string) {.task(kind=no_rts, stoppable=false).} =
           lightpushHandler)
       else:
         asyncSpawn wakuNode.publish(DefaultTopic, wakuMessage, conf.rlnRelay)
+
+proc getCustomTokens*() {.task(kind=no_rts, stoppable=false).} =
+  let tokens = status.getCustomTokens()
```

Comment:
Probably should check logged in state just like the other two tasks, as  it is required on the nim-status client side. We can keep it as it is, but there would be an inconsistent message to the end user, which may be confusing.

Reply:
Done.